tests/test_unit/test_cli/test_build_deploy_snapshots/cdf_inrobot_common.yaml

```diff
@@ -154,7 +154,7 @@ Function:
   fileId: -1
   functionPath: ./handler.py
   metadata:
-    cdf-toolkit-function-hash: 551b03bfdfe83657d6eee7f45fd8b7efb9e30c25797b01e7d5e87251f8fa7bfc
+    cdf-toolkit-function-hash: 0603a6565ce71cd41e0f8959c2b271827d4ba50491c14f2ac78a8d00bfcffd9c
```

Comment:
I think this is because in the temporary way of setting up modules, I do not copy over `README.md` files.